tests/core/methods/test_matrix_exponential.py

```
@@ -120,19 +66,125 @@ def test_expm_krylov_smaller_subspace() -> None:
+    res = expm_krylov(mock_op, v, dt)
+
+    np.testing.assert_array_equal(res, v)
+    mock_op.assert_not_called()
+
+
+def test_expm_krylov_numba_execution() -> None:
+    """Test execution path when Numba is enabled (large vector)."""
+    # Verify Numba module is available
+    # Verify Numba module is available
+    pytest.importorskip("mqt.yaqs.core.methods.lanczos_numba")
+    from mqt.yaqs.core.methods.lanczos_numba import orthogonalize_step  # noqa: PLC0415
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)